docs/aws-ec2/README.md

```diff
@@ -0,0 +1,22 @@
+# EC2 Developer Setup
+
+## Background
+
+For easy development purposes, it is beneficial to setup a ec2 instance for development as the memory requirements for armada can be quite intense.  For example, a mac M1 with 8 gb is too small for development.  Some images will also not be supported on a M1 so it is beneficial to setup a ec2 instance for development.
+
+## Instructions
+
+- We suggest a t3.xlarge instance from aws ec2 with Amazon Linux.  16 GB of memory is suggested.
+- During selection of instance, Add a large volume to your ec2 instance (100 gb)
+- During selection of instance, add Security Group Rules that allows you to access tcp ports from your desktop browser.  This is useful for viewing lookout UI or other web services
```

Comment:
Recommend rephrase:
When selecting the instance, you will have the opportunity to choose a security group. You may need to make a new one. Be sure to add a rule allowing inbound communication on port 22 so that you can access your server via SSH. We recommend that you restrict access to the IP address from which you access the Internet, or a small CIDR block containing it.

If you want to use your browser to access Armada Lookout UI or other web-based interfaces, you will also need to grant access to their respective ports. For added security, consider [using an SSH tunnel](https://www.ssh.com/academy/ssh/tunneling/example) from your local machine to your development server instead of opening those ports.

Reply:
Just a pro tip, you can add suggestions to PRs which would allow a reviewer to substitute the code or text.  Its useful for cases where you have simple fixes to suggest.

Added this to doc.